projects/Mallard/src/services/errors.ts

```diff
@@ -53,16 +69,52 @@ class ErrorServiceImpl implements ErrorService {
                 __DEV__ ? 'DEV' : isInBeta() ? 'BETA' : 'RELEASE',
             )
             Sentry.setExtra('react', true)
-            this.hasConfigured = true
+            this.hasSentryConfigured = true
         }
 
         while (this.pendingQueue.length > 0) {
             const err = this.pendingQueue.pop()
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            Sentry.captureException(err!)
+            if (err) {
+                Sentry.captureException(err)
+                this.crashlytics.recordError(err)
+            }
+        }
+    }
+
+    private async initCrashlytics() {
+        const defVal = this.crashlytics.isCrashlyticsCollectionEnabled
+        console.log('Crashlytics current status:', defVal)
```

Comment:
is it deliberate to leave this console.log in?

Reply:
I have been informed by @jimhunty that these logs aren't available in prod build. So it should be fine and help us monitor/debug things locally.